packages/oui-field/src/field.html

```diff
@@ -23,7 +23,7 @@
         ng-if="$ctrl.isErrorVisible()">
         <span ng-messages="$ctrl.getFirstError()" role="alert">
             <span class="oui-icon oui-icon-error" aria-hidden="true"></span>
-            <span data-ng-repeat="errorKey in $ctrl.getMessagesOrder() track by $index"
+            <span data-ng-repeat="errorKey in $ctrl.getMessagesOrder()"
```

Comment:
Why ?

Reply:
It doesn't refresh the error, and displayed nothing